packages/menu-bar/test/overflow.test.js

```diff
@@ -160,22 +143,6 @@ describe('overflow', () => {
       await nextResize(menu);
 
       expect(buttons[0].getAttribute('tabindex')).to.equal('0');
-      expect(buttons[1].getAttribute('tabindex')).to.equal('-1');
-    });
-
-    it('should set tabindex -1 on the overflow menu in tab navigation', async () => {
```

Comment:
This is a copy-paste of the test above. With `tabNavigation` the logic for setting `tabindex="0"` on the buttons that remain visible is not needed since all enabled buttons have `tabindex="0"` by default. Removed the test.